dataland-backend/src/main/kotlin/org/dataland/datalandbackend/repositories/utils/StoredCompanySearchFilter.kt

```diff
@@ -5,12 +5,12 @@ package org.dataland.datalandbackend.repositories.utils
  * convenient usage of SEPL instructions in the query
  */
 data class StoredCompanySearchFilter(
-    val dataTypeFilter: List<String>,
-    val countryCodeFilter: List<String>,
-    val sectorFilter: List<String>,
-    val searchString: String,
-    val nameOnlyFilter: Boolean,
-    val uploaderIdFilter: List<String>,
+    val dataTypeFilter: List<String> = listOf(),
+    val countryCodeFilter: List<String> = listOf(),
+    val sectorFilter: List<String> = listOf(),
+    val searchString: String = "",
+    val nameOnlyFilter: Boolean = false,
+    val uploaderIdFilter: List<String> = listOf(),
```

Comment:
Wie konnte das eigentlich vorher ohne default-Werte funktionieren?
Die size der Listen wurde doch in der folgenden Query (die es ja in main bereits gibt) ständig abefragt:

![image](https://github.com/d-fine/Dataland/assets/99255296/6e31c192-8f7a-401c-a03e-0c6ffd15c765)